build/images/base/build.sh

```diff
@@ -129,34 +135,31 @@ if $PULL; then
     done
 fi
 
-docker build $PLATFORM_ARG --target cni-binaries \
-       --cache-from antrea/cni-binaries:$CNI_BINARIES_VERSION \
-       -t antrea/cni-binaries:$CNI_BINARIES_VERSION \
-       --build-arg CNI_BINARIES_VERSION=$CNI_BINARIES_VERSION \
-       --build-arg BUILD_TAG=$BUILD_TAG .
+function docker_build_and_push() {
```

Comment:
This looks similar to the one in ovs/build.sh, maybe they can be refined and moved to build-utils.sh?

Reply:
I had thought about it originally, and I don't feel strongly either way, but it felt to me like the script was more readable this way, rather than have to pass Docker build args as a function argument.